products/extensions/b2b-components/customer-specific-features/feature-toggles.md

```diff
@@ -6,14 +6,14 @@ nav:
 ---
 
 ## Introduction
-A new "Customer Specific Features" section on the Customer detail page allows the shop merchant to turn B2B features on or off for a specific customer. This section aims to provide each customer with their own set of specific features, granting them access to certain B2B components within the shop.
+A new "Customer-specific features" section on the Customer detail page allows the shop merchant to turn B2B features on or off for a specific customer. This section aims to provide each customer with their own set of specific features, granting them access to certain B2B components within the shop.
 
 ![Feature Toggles](../../../../assets/b2b-feature-toggles.png)
```

Comment:
Can you please double check that those relative links for `assets` is correct? Especially in files where the level of nesting was changed.

Reply:
@bojanrajh yes it was the only asset as far as I saw. You can see it in the proof vid :slightly_smiling_face: 

![image](https://github.com/shopware/docs/assets/8600299/935bf882-821d-43cc-a980-9991208ce8e5)